datajs/src/browser-utils/utils.js

```diff
@@ -1,15 +1,16 @@
-const stream = require('stream')
+import { Readable } from 'stream'
+// import { File } from '../file-base'
```

Comment:
Why is this commented out? If we don't need it let's delete it.

Reply:
Not needed. I'll delete it